Formula/edge-impulse-cli.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class EdgeImpulseCli < Formula
+  desc "Command-line interface tools for Edge Impulse"
+  homepage "https://github.com/edgeimpulse/edge-impulse-cli#readme"
```

Comment:
The readme is already on this page
```suggestion
  homepage "https://github.com/edgeimpulse/edge-impulse-cli"
```

Reply:
Thank you, removed anchor